### PR TITLE
app now working on this branch

### PR DESCRIPTION
**Pull request by Laura-D-Andrews**
TL;DR: 
I wasn't paying attention and messed up a merge into the main branch while trying to make a quick fix and deleted the model attributes Alain and I had put in for FE and got our migrations files messed up. 

This version of the app seems to be bug free and runs well through my local in insomnia. I reverted it back to the last working version we pushed and then re-added the colors, borders, etc model attributes FE was asking for. I had to delete some migration files and re-migrate to get my database working. The cards_sent and cards_received related_names are also fixed in views (we had changed them on the models with Amy but not in views) and those endpoints are also working in insomnia through my local DB. Those endpoints are not working through our production database right now because the last time we pushed on main we hadn't corrected the related names. Per FE all other endpoints are working through Render. 

I did have to delete my local database.....so I'm a bit worried we are gonna have to delete our production database. We don't have much data in it though. 

We can talk about merging this branch with the main branch and then see what's going on in Render after it's pushed.

### CODE REVIEW

**Comment by alainishimwe** [comment]
Hey Laura, I am ok with merging this branch with the main branch and see how it's gonna affect Render.
